core/image.cpp

```diff
@@ -880,8 +882,8 @@ void Image::resize_to_po2(bool p_square) {
 void Image::resize(int p_width, int p_height, Interpolation p_interpolation) {
 
 	ERR_FAIL_COND_MSG(data.size() == 0, "Cannot resize image before creating it, use create() or create_from_data() first.");
-
 	ERR_FAIL_COND_MSG(!_can_modify(format), "Cannot resize in compressed or custom image formats.");
+	ERR_FAIL_COND_MSG(write_lock.ptr(), "Image must not be locked during resize.");
```

Comment:
I prefer language more like what you have below. I.e. "image cannot be resized when it is locked"

Same goes for the above comment. 

The current language creates confusion because locked is both a state and an action. So it is ambiguous whether you are telling the user not to lock the image when a resize is occuring, or whether you are telling the user that a locked image cannot be resized.

Reply:
Updated. I changed it to "Cannot resize image when it is locked" to match the above error message "Cannot resize image before creating it".